docs/content/gds/admin/configuration/bff.en.md

```diff
@@ -1,7 +1,172 @@
+
+The primary database store is configured as follows:
+
+| EnvVar                       | Type   | Default | Description                                                                          |
+|------------------------------|--------|---------|--------------------------------------------------------------------------------------|
+| GDS_BFF_DATABASE_URL             | string |         | Required, the DSN to connect to the database on (see below for details)              |
+| GDS_BFF_DATABASE_REINDEX_ON_BOOT | bool   | false   | When the server starts, instead of loading indexes from disk, recreate and save them |
+| GDS_BFF_DATABASE_INSECURE        | bool   | false   | If set do not connect to the TrtlDB with mTLS authentication                         |
+| GDS_BFF_DATABASE_CERT_PATH       | string |         | The path to the mTLS client-side certs for database auth                             |
+| GDS_BFF_DATABASE_POOL_PATH       | string |         | The path to the mTLS public cert pools to accept server connections                  |
+
+The `GDS_BFF_DATABASE_URL` is a standard DSN with a scheme, host, path, and query parameters. In the case of the BFF the scheme can be either `trtl://` or `leveldb://`.
+
+When connecting to a TrtlDB, the host and port need to be specified with a trailing slash, e.g. `trtl://localhost:4436/`. Connecting via mTLS is only relevant when connecting to a TrtlDB. If `GDS_BFF_DATABASE_INSECURE` is `false`, then the `GDS_BFF_DATABASE_CERT_PATH` and `GDS_BFF_DATABASE_POOL_PATH` are required.
+
+When connecting to a LevelDB, the path to the directory on disk where the leveldb should be stored must be passed to the DSN. To specify a relative path, use three slashes: `leveldb:///relpath/to/db`, to specify an absolute path use four: `leveldb:////abspath/to/db`.
+
+### Email and SendGrid
+
+The BFF uses [SendGrid](https://sendgrid.com/) to send email notifications and to enable communication workflows with users and admins. Configure BFF to use SendGrid as follows:
+
+| EnvVar                | Type   | Default                                           | Description                                                              |
+|-----------------------|--------|---------------------------------------------------|--------------------------------------------------------------------------|
+| GDS_BFF_SERVICE_EMAIL | string | TRISA Directory Service <admin@vaspdirectory.net> | The email address used as the sender for all emails from the BFF system. |
+| SENDGRID_API_KEY      | string |                                                   | API Key to authenticate to SendGrid with.                                |
+| GDS_BFF_EMAIL_TESTING | bool   | false                                             | Use email in testing mode rather than send live emails                   |
```

Comment:
```suggestion
| GDS_BFF_EMAIL_TESTING | bool   | false                                             | Use email in testing mode rather than send live emails.                   |
```